edge-util/src/Microsoft.Azure.Devices.Edge.Util/metrics/prometheus.net/MetricsProvider.cs

```diff
@@ -66,10 +69,46 @@ List<string> GetLabelNames(List<string> labelNames)
             var allLabelNames = new List<string>
             {
                 MetricsConstants.IotHubLabel,
-                MetricsConstants.DeviceIdLabel
+                MetricsConstants.DeviceIdLabel,
+                "instance_number"
             };
             allLabelNames.AddRange(labelNames);
             return allLabelNames;
         }
+
+        /// <summary>
+        /// Reads or creates a file to determine how many times this module has been restarted.
+        /// This is because prometheous metrics are cleared on restart.
+        /// Includung an instance number makes it clear when a counter reset.
+        /// If the time series of metrics goes
+        ///     {instance=1) 500
+        ///     {instance=1) 600
+        ///     {instance=1) 700
+        ///     {instance=2) 90
+        ///     {instance=2) 190
```

Comment:
The timestamp should also be logged here as well. 
That way you can do a timeline of restarts along with other module logs.

Reply:
This isn't intended as a logging feature, more just alerting that data was lost. Timestamps are more appropriate for logs. I might take this out, depending on feedback in meeting today

Reply:
If I understand correctly, this should provide a metric on the edgeAgent availability, right? 
if not, what's a mechanism that we use to track the edgeAgent availability?